### PR TITLE
AN-7981 Replace all instances of "student" in the text with "learner"

### PR DESCRIPTION
**Pull request by thallada**
Big diff! But, this is all just a simple replace.

I only did more than that in one place where a simple replace resulted in the awkward phrase "supporting learner learning", so I made it just "supporting learners".

### CODE REVIEW

File: analytics_dashboard/static/js/enrollment-geography-main.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ require(['vendor/domReady!', 'load/init-page'], function(doc, page) {
                     model: page.models.courseModel,
                     modelAttribute: 'enrollmentByCountry',
                     // eslint-disable-next-line max-len
-                    tooltip: gettext('Student location is determined from IP address. This map shows where students most recently connected.')
+                    tooltip: gettext('Learner location is determined from IP address. This map shows where learners ' +
```
Comment (dsjen) [inline analytics_dashboard/static/js/enrollment-geography-main.js]: This is super annoying, but the [recommendation](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html#do-not-append-strings-or-interpolate-values) is for this to be a single long line in order to properly harvest the string for translators.

**Review by dsjen — COMMENTED** [review]
I left a note about translation harvesting.  Otherwise, LGTM! 👍

**Comment by thallada** [comment]
@dsjen good to merge now?

**Comment by dsjen** [comment]
Thanks! 👍

**Pull request merged**